### PR TITLE
[vcpkg] Rename the msbuild property VcpkgRoot

### PR DESCRIPTION
**Pull request by BillyONeal**
… to VcpkgCurrentInstalledDir, and set VcpkgRoot to the expected root instead.

### CODE REVIEW

File: ports/qt5-webengine/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 message(STATUS "${PORT} requires a lot of free disk space (>300GB), ram (>32 GB) and time (>4h per configuration) to be successfully build.\n\
 -- As such ${PORT} is not properly tested.\n\
 -- If ${PORT} fails post build validation please open up an issue. \n\
--- If it fails due to post validation the successfully installed files can be found in <vcpkgroot>/packages/${PORT}_${TARGET_TRIPLET} \n\
--- and just need to be copied into <vcpkgroot>/installed/${TARGaET_TRIPLET}")
+-- If it fails due to post validation the successfully installed files can be found in ${CURRENT_PACKAGES_DIR} \n\
```
Comment (BillyONeal) [inline ports/qt5-webengine/portfile.cmake:6]: This is a drive-by mistake I found since I grepped for 'VcpkgRoot'.

File: toolsrc/src/vcpkg/export.cpp [hunk 2]
```diff
@@ -61,9 +61,6 @@ namespace vcpkg::Export
     {
         return Strings::format(R"###(
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```
Comment (strega-nil) [inline toolsrc/src/vcpkg/export.cpp:64]: Is there a reason we don't have to export this?

**Review by strega-nil — APPROVED** [review]
Also, were there really no other uses of `VcpkgRoot` anywhere?

**Comment by BillyONeal** [comment]
> Also, were there really no other uses of `VcpkgRoot` anywhere?

Correct.

File: toolsrc/src/vcpkg/export.cpp (hunk 2 above)
Comment (BillyONeal) [reply toolsrc/src/vcpkg/export.cpp:64]: The file imported is `$/scripts/buildsystems/msbuild/vcpkg.targets` so the vcpkg root is already known from that. It might be an optimization to set the path here but I don't want to significantly change function if I don't have to.
